server/lib/resolvers/resolverMapper.js

```diff
@@ -33,14 +33,32 @@ module.exports = function (dataSources, resolverMappings) {
     let builder = resolverBuilders[dataSource.type]
 
     if (builder) {
```

Comment:
Here shows that this method is missing encapsulations. Note that the complexity to understand it is high and it is doing many actions.